### PR TITLE
Mid-August deploys (Engine)

### PR DESCRIPTION
**Pull request by jakerockland**
## Description of the change

Deployments for the following:

(goerli)
- https://github.com/ArtBlocks/artblocks/issues/1364
- https://github.com/ArtBlocks/artblocks/issues/1357
- https://github.com/ArtBlocks/artblocks/issues/1340

(mainnet)
- https://github.com/ArtBlocks/artblocks/issues/1341
- https://github.com/ArtBlocks/artblocks/issues/1358

### CODE REVIEW

File: packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENTS.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+**Minters:**
+
+**MinterSetPriceV4:** https://etherscan.io/address/0x936130f3042EE825d375A3beeDA04A2bA4880197#code
+
+
+
+**Metadata**
+
+- **Starting Project Id:** 0
+- **Token Name:** GAZELL.iO
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENTS.md:72]: quick check - is this the desired casing for token name? (all caps, except the `i`?)

File: packages/contracts/deployments/engine/V3/partners/vertu-flex/DEPLOYMENT_LOGS.log [hunk 2]
```diff
@@ -66,3 +66,55 @@ Contracts metadata upsert input:
...
+Contracts metadata upsert input:
+{
+  "address": "0xedd5c3d8e8fc1e88b93a98282b8ccfd953c483a4",
+  "name": "Vertu Fine Art",
+  "bucket_name": "vertu-fine-art-mainnet",
+  "default_vertical_name": "flex"
+}
+Successfully upserted 1 contract
+[ACTION] provider primary and secondary sales payment addresses remain as deployer addresses: 0xB8559AF91377e5BaB052A4E9a5088cB65a9a4d63. Update later as needed.
+[ACTION] AdminACL's superAdmin address is 0x57F1C8d52baFCbDDCAaa71BE11FEA6E48e11af11, don't forget to update if requred.
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/partners/vertu-flex/DEPLOYMENT_LOGS.log:120]: smol reminder of these follow-on actions 👍

File: packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENT_LOGS.log [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Contracts metadata upsert input:
+{
+  "address": "0xa86cd4ecebd96085fce4697614d30600803455c4",
+  "name": "GAZELL.iO",
+  "bucket_name": "gazell-io-mainnet",
+  "default_vertical_name": "flex"
+}
+Successfully upserted 1 contract
+[ACTION] provider primary and secondary sales payment addresses remain as deployer addresses: 0xB8559AF91377e5BaB052A4E9a5088cB65a9a4d63. Update later as needed.
+[ACTION] AdminACL's superAdmin address is 0xFf90F956C9A7d52aC21C37fe7a34801849887f19, don't forget to update if requred.
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENT_LOGS.log:84]: smol reminder of these follow-on actions 👍

**Review by ryley-o — APPROVED** [review]

File: packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENTS.md (hunk 1 above)
Comment (ryley-o) [reply packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENTS.md:72]: okay, looks legit based on quick google search. 👍

Comment (jakerockland) [reply packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENTS.md:72]: Yezzzer. :)

File: packages/contracts/deployments/engine/V3/partners/vertu-flex/DEPLOYMENT_LOGS.log (hunk 2 above)
Comment (jakerockland) [reply packages/contracts/deployments/engine/V3/partners/vertu-flex/DEPLOYMENT_LOGS.log:120]: Interesting – I think this is actually spurious and due to an error in our action message logic, which I've opened up https://github.com/ArtBlocks/artblocks-contracts/pull/959 to address.

Could you please take a look and let me know if I'm trippin' here?

File: packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENT_LOGS.log (hunk 3 above)
Comment (jakerockland) [reply packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENT_LOGS.log:84]: Interesting – I think this is actually spurious and due to an error in our action message logic, which I've opened up https://github.com/ArtBlocks/artblocks-contracts/pull/959 to address.

Could you please take a look and let me know if I'm trippin' here?

Comment (ryley-o) [reply packages/contracts/deployments/engine/V3/partners/gazelli-flex/DEPLOYMENT_LOGS.log:84]: nice catch, thank you!

**Pull request merged**